rkt/enter.go

```diff
@@ -17,43 +17,44 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"io/ioutil"
 
 	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/schema"
 	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/schema/types"
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/spf13/cobra"
 	"github.com/coreos/rkt/common"
 	"github.com/coreos/rkt/stage0"
 	"github.com/coreos/rkt/store"
 )
 
 var (
-	cmdEnter = &Command{
-		Name:    cmdEnterName,
-		Summary: "Enter the namespaces of an app within a rkt pod",
-		Usage:   "[--imageid IMAGEID] UUID [CMD [ARGS ...]]",
-		Run:     runEnter,
-		Flags:   &enterFlags,
-	}
-	enterFlags     flag.FlagSet
-	flagAppImageID types.Hash
+	cmdEnter = &cobra.Command{
+		Use:   "enter [--imageid IMAGEID] UUID [CMD [ARGS ...]]",
```

Comment:
Since spf13/pflag#20 is not merged and for consistency we can try to write all the examples with the equals sign so they actually work.

```
Use:   "enter [--imageid=IMAGEID] UUID [CMD [ARGS ...]]"
```